caikit/core/model_manager.py

```diff
@@ -180,28 +189,96 @@ def train(
         # Return a handle to the training
         return model_future
 
+    def start_prediction_job(
+        self,
+        model: ModuleBase,
+        prediction_func_name: str,
+        *args,
+        predictor: Union[str, JobPredictorBase] = "default",
+        wait: bool = False,
+        **kwargs,
+    ) -> JobFutureBase:
+        """Start a prediction job using a job_predictor.
+
+        Args:
+            model (ModuleBase): Loaded model to run prediction on
+            prediction_func_name (str): String reference to name of function to run
+            predictor (Union[str, JobPredictorBase], optional): Which job_predictor to use.
+              Defaults to "default".
+            wait (bool, optional): Weather to wait for job to finish. Defaults to False.
+
+        Returns:
+            JobFutureBase: Future to track job result
+        """
+        error.type_check("<COR02418775E>", ModuleBase, model=model)
+
+        # Get the predictor to use
+        inferencer: JobPredictorBase = self.get_predictor(predictor)
+
+        # Start the prediction job
+        with alog.ContextTimer(log.debug, "Started prediction job in: "):
+            model_future = inferencer.predict(
+                model,
+                prediction_func_name,
+                *args,
+                **kwargs,
+            )
+            log.debug(
+                "Started Prediction Job %s",
+                model_future.id,
+            )
+
+        # If requested, wait for the future to complete
+        if wait:
+            log.debug("Waiting for prediction %s to complete", model_future.id)
+            with alog.ContextTimer(
+                log.debug, "Finished training %s in: ", model_future.id
+            ):
+                model_future.wait()
+
+        # Return a handle to the future
+        return model_future
+
     def get_model_future(
         self,
-        training_id: str,
-    ) -> ModelTrainerBase.ModelFutureBase:
+        future_id: str = None,
```

Comment:
I think that the support for fetching prediction jobs should be a separate function since `ModelFuture` is strictly for training